tests/e2e/specs/LoadComplexData.spec.js

```diff
@@ -16,13 +16,10 @@ describe('Validation Rules (Advanced test)', () => {
     cy.assertPreviewData({
       'FPP_COLLEGE': '',
       'FPP_DEPARTMENT': '',
-      // 'FPP_USER_FULL_TIME': {
-      //   'FPP_TL_FULL_TIME': null,
-      //   'FPP_NTL_FULL_TIME': null,
-      // },
-      // @todo this was overwritten to null by SelectList when list is refreshed
-      // at FormSelectList.vue @ updateWatcherDependentFieldValue()
-      'FPP_USER_FULL_TIME': null,
+      'FPP_USER_FULL_TIME': {
```

Comment:
This should not be changed, because it was fixed by a previous PR (not yet merged): https://github.com/ProcessMaker/screen-builder/pull/1303